README.md

```diff
@@ -392,7 +392,7 @@ instance = SingleVMOpenEdXInstance.objects.create(
     sub_domain='dogwood.sandbox',
     name='Dogwood',
     fork_name='edx/edx-platform',
-    branch_name='named-release/dogwood',
+    branch_name='named-release/dogwood.rc',
```

Comment:
@antoviaque I guess `branch_name` is not the best name when it can be something other than branch... Maybe we should think about renaming it to `ref_name`. Or perhaps use a separate property `tag_name` for when build is based on a tag rather than a branch?